crucible/src/Lang/Crucible/Backend/ProofGoals.hs

```diff
@@ -266,22 +268,35 @@ gcAddAssumes as' (CollectingAssumptions as gls) = CollectingAssumptions (as <> a
 gcAddAssumes as' gls = CollectingAssumptions as' gls
 
 {- | Pop to the last push, or all the way to the top, if there were no more pushes.
-If the result is 'Left', then we popped until an explicitly marked push;
-in that case we return:
-
-    1. the frame identifier of the popped frame,
-    2. the assumptions that were forgotten,
-    3. any proof goals that were generated since the frame push, and
-    4. the state of the collector before the push.
-
+If the result is 'Left', then we popped until an explicitly marked push.
 If the result is 'Right', then we popped all the way to the top, and the
 result is the goal tree, or 'Nothing' if there were no goals. -}
```

Comment:
I think these Haddocks were meant to annotate `gcPop`, but now they no longer do so since the definitions of `PoppedFrame`/`collectPoppedGoals` were introduced. Can you move these Haddocks down to be next to `gcPop`?

Reply:
Oh, whoops! Yep.